.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.2
```

Comment:
Why the switch here?

Reply:
we claim to support 2.2, but rubocop was looking for 2.3 syntax.